spec/models/post_action_spec.rb

```diff
@@ -987,4 +987,38 @@ def limiter(tl, type)
       end
     end
   end
+
+  describe "count_per_day_for_type" do
+    before { PostActionCreator.create(Fabricate(:user), post, :like) }
```

Comment:
Could speed this up slightly by fabricating the user before, outside of the `before` block